Dockerfile

```diff
@@ -11,55 +33,76 @@ RUN ln -snf /bin/bash /bin/sh
+RUN mkdir -p /home/rmguser/Code
 
-# Set Bash as the default shell for following commands
-SHELL ["/bin/bash", "-c"]
+# Change working directory to Code
+WORKDIR /home/rmguser/Code
 
-# cd
-WORKDIR /rmg
+# Clone the RMG base and database repositories. The pulled branches are only the main branches.
+RUN git clone -b main https://github.com/ReactionMechanismGenerator/RMG-Py.git \ 
+    && git clone -b main https://github.com/ReactionMechanismGenerator/RMG-database.git
 
-# Clone the RMG base and database repositories
-RUN git clone -b main https://github.com/ReactionMechanismGenerator/RMG-Py.git
-RUN git clone -b main https://github.com/ReactionMechanismGenerator/RMG-database.git
+# cd into RMG-Py
+WORKDIR /home/rmguser/Code/RMG-Py
 
-# build the conda environment
-WORKDIR /rmg/RMG-Py
-RUN conda env create --file environment.yml
+# Install RMG-Py and then clean up the micromamba cache
+RUN micromamba create -y -f environment.yml && \
+    micromamba clean --all -f -y
 
-# This runs all subsequent commands inside the rmg_env conda environment
-#
-# Analogous to just activating the environment, which we can't actually do here
-# since that requires running conda init and restarting the shell (not possible
-# in a Dockerfile build script)
-SHELL ["conda", "run", "--no-capture-output", "-n", "rmg_env", "/bin/bash", "-c"]
+# Activate the RMG environment
+ARG MAMBA_DOCKERFILE_ACTIVATE=1
+ENV ENV_NAME=rmg_env
 
-# Set environment variables as directed in the RMG installation instructions
-ENV RUNNER_CWD=/rmg
-ENV PYTHONPATH="$RUNNER_CWD/RMG-Py:$PYTHONPATH"
-ENV PATH="$RUNNER_CWD/RMG-Py:$PATH"
+# Set environment variables
+# These need to be set in the Dockerfile so that they are available to the build process
+ENV PATH /opt/conda/envs/rmg_env/bin:$PATH
+ENV PYTHONPATH /home/rmguser/Code/RMG-Py:$PYTHONPATH
+ENV PATH /home/rmguser/Code/RMG-Py:$PATH
 
 # Build RMG
-RUN make
+RUN make \
+    && echo "export PYTHONPATH=/home/rmguser/Code/RMG-Py" >> ~/.bashrc \
+    && echo "export PATH=/home/rmguser/Code/RMG-Py:$PATH" >> ~/.bashrc
 
-# Install and link Julia dependencies for RMS
-RUN python -c "import julia; julia.install(); import diffeqpy; diffeqpy.install()" || true
-RUN julia -e 'using Pkg; Pkg.add(PackageSpec(name="ReactionMechanismSimulator",rev="main")); using ReactionMechanismSimulator' || true 
+# Create the link between Julia and RMG-Py. A symbolic link is created between Python-JL and Python.
+# We do not use the command in Julia 'using ReactionMechanismSimulator' because the command is only for determining if the package is installed.
+# The command does not actually install the package but instead increases the build time, therefore not required.
+# Install RMS
+# The extra arguments are required to install PyCall and RMS in this Dockerfile. Will not work without them.
+# Final command is to compile the RMS during Docker build - This will reduce the time it takes to run RMS for the first time
+RUN touch /opt/conda/envs/rmg_env/condarc-julia.yml
```

Comment:
So there should be a commit that mentions this. I spoke with the micromamba developer and he suggested this because Julia wasn't playing nice with micromamba anymore after we changed from the rmg channel to the Julia channel